google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1/StreamWriterTest.java

```diff
@@ -304,6 +306,38 @@ public void testAppendSuccessAndInStreamError() throws Exception {
     writer.close();
   }
 
+  @Test
```

Comment:
Add another test case on the new error handling you added on onDone.

Reply:
Done